cmd/ingester/main.go

```diff
@@ -91,21 +91,19 @@ func main() {
 			consumer.Start()
 
 			hc.Ready()
-			select {
-			case <-signalsChannel:
-				logger.Info("Jaeger Ingester is starting to close")
-				err := consumer.Close()
+			<-signalsChannel
+			logger.Info("Jaeger Ingester is starting to close")
```

Comment:
nit: s/close/shutdown

Reply:
Sure. I can change the message too. Was just focusing on the issues gosimple pointed out.

Reply:
As a follow up, can we get rid of service name here? I don't see how this is more descriptive when the service logs are always kept separate. Even crossdock prefixes service names to logs.

Reply:
I thought the same but didn't want to look too pedantic. Yes the name is unnecessary.